src/electron/menu.ts

```diff
@@ -43,10 +43,6 @@ export function getTemplateFromKeymap(
                 label: 'Add Cloud Space',
```

Comment:
I would personally remove any `cloud` mention from the application. 

`Add Space`